### ISSUE TITLE
Do you have the generator weights?

### ISSUE THREAD

**Issue by adar-cohen-imagenai** [gfxdisp/mdf#10]
Hey, thank you for this project, well done!
Do you mind sharing your generator checkpoint (those that were trained based on the MDF loss)?
I want to simply run inference on the JPEG artifacts removal and see the results

**Comment by aamir-mustafa** [comment]
Hi, Thanks for your interest in our work. Please note that the discriminator checkpoints are used as feature extractors for training the jpeg artefacts removal model, instead of conventional loss functions. The underlying model does not change. 

Hope that helps.

**Comment by adar-cohen-imagenai** [comment]
@aamir-mustafa Hey Amir, thanks for your reply!
I totally understand, what I would like, is to have the generator model checkpoint, the one you train based on the MDF loss.

In your paper, you show examples of a few outputs, each output is some model based on some loss function. The model that was trained based on your MDF loss, reached the highest results. Im interested in that checkpoint of the generator, and not the discriminatory that calc the MDF loss.

**Comment by delyan-boychev** [comment]
@adar-cohen-imagenai Hello, Adar! I have come up with the same issue. You can check out the original repository of SinGAN - https://github.com/tamarott/SinGAN. The implementation of the generator and the discriminator are the same. You can change the perturbations added to the input in the file SinGAN/functions.py. After that, you can reproduce the results using the proposed perturbations in the paper.
